doc/man3/SSL_CTX_set_client_hello_cb.pod

```diff
@@ -68,6 +70,16 @@ in the ClientHello.  B<*outlen> contains the number of elements in the array.
 In situations when the ClientHello has no extensions, the function will return
 success with B<*out> set to NULL and B<*outlen> set to 0.
 
+SSL_client_hello_get_raw_extensions() is similar with
+SSL_client_hello_get1_extensions_present(), without internal memory allocation.
+The B<exts> parameter can be NULL to simply get the extensions length
```

Comment:
Replace `extensions length` with `number of extensions`.

Reply:
done.